annotation-processing/src/main/java/com/exactpro/epfast/annotation/processing/FieldSetterNameGenerator.java

```diff
@@ -0,0 +1,21 @@
+package com.exactpro.epfast.annotation.processing;
+
+import java.nio.charset.StandardCharsets;
+import java.security.MessageDigest;
+import java.security.NoSuchAlgorithmException;
+
+public class FieldSetterNameGenerator {
+    public static String generateClassName(String fastTypeName) throws NoSuchAlgorithmException {
+        MessageDigest digest = MessageDigest.getInstance("SHA-256");
+        byte[] hash = digest.digest(fastTypeName.getBytes(StandardCharsets.UTF_8));
+        StringBuilder hexString = new StringBuilder("A");
```

Comment:
`"a" + Base64.getEncoder().withoutPadding().encodeToString(hash)`

It's good idea to store encoder and reuse it.